src/operations.json

```diff
@@ -0,0 +1,17 @@
+{
+	"insert-ai-generated-content": {
+		"label": "Insert",
+		"tooltip": "Insert the generated content into the document",
+		"steps": [
+			{
+				"type": "transform/getSelectionInDocument"
```

Comment:
If this naming should be consistent with the platform. then it should be called something like:
setOverrideRangeToSelection